### PR TITLE
dcrtime: -h is host or host:port only

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

**Comment by jcvernaleo** [comment]
On testnet anyway this works for me (both the default server and a local one).

**Comment by marcopeereboom** [comment]
I like this.

File: cmd/dcrtime/dcrtime.go [hunk 1]
```diff
@@ -42,6 +42,16 @@ var (
...
+// normalizeAddress returns addr with the passed default port appended if
+// there is not already a port specified.
+func normalizeAddress(addr, defaultPort string) string {
+	_, _, err := net.SplitHostPort(addr)
+	if err != nil {
+		return net.JoinHostPort(addr, defaultPort)
+	}
+	return addr
+}
+
```
Comment (marcopeereboom) [inline cmd/dcrtime/dcrtime.go]: Isn't this code already in config.go?

Comment (dajohi) [inline cmd/dcrtime/dcrtime.go]: dcrtimed's config.

**Pull request merged**